frontend/src/common/enums/app/env.enum.ts

```diff
@@ -1,7 +1,8 @@
-const { REACT_APP_API_ORIGIN_URL } = process.env;
+const { REACT_APP_API_ORIGIN_URL, REACT_APP_SOCKET_SERVER } = process.env;
 
 const ENV = {
   API_PATH: REACT_APP_API_ORIGIN_URL ?? '',
+  SOCKET_URL: REACT_APP_SOCKET_SERVER ?? '',
```

Comment:
```suggestion
  SOCKET_URL: REACT_APP_SOCKET_SERVER ?? '',
```
try to avoid an additional ENV. should be a part of `api`